aip_xx1_description/urdf/sensor_kit.xacro

```diff
@@ -197,6 +197,36 @@
       height="400"
       fov="1.3"
     />
+    <xacro:monocular_camera_macro
+      name="camera6/camera"
+      parent="sensor_kit_base_link"
+      namespace=""
+      x="${calibration['sensor_kit_base_link']['camera6/camera_link']['x']}"
+      y="${calibration['sensor_kit_base_link']['camera6/camera_link']['y']}"
+      z="${calibration['sensor_kit_base_link']['camera7/camera_link']['z']}"
```

Comment:
camera7/camera_link -> camera6/camera_link?

Reply:
I will address it in the new repo